numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -233,12 +233,19 @@ def compile(src, name, cc):
     #   being optimized away.
     major, minor = cc
     arch = f'--gpu-architecture=compute_{major}{minor}'
-    include = f'-I{config.CUDA_INCLUDE_PATH}'
+
+    cuda_include = [
+        f"-I{get_cuda_paths()['include_dir'].info}",
+        f"-I{config.CUDA_INCLUDE_PATH}"
```

Comment:
I think the logic here might need a bit more refinement. I'm thinking about the situation where a user has set `config.CUDA_INCLUDE_PATH`, and has a system toolkit in the default location, but does not have a toolkit installed via conda. In this case, I think the toolkit in the system default location will always be preferred, because it will come first on the path.

Maybe it would simplify things if there were only two ways to find the includes:

- The conda environment (added in this PR)
- From `config.CUDA_INCLUDE_PATH` (already existing)

and not add new functionality to try to find the toolkit from the system default path in this PR?

Reply:
Ah, this makes sense. The config default is the same as system default. So when we don't find conda CTK, we fall back to system default, which is fair solution.

Reply:
This is changed in [1dcfd78](https://github.com/NVIDIA/numba-cuda/pull/17/commits/1dcfd78cd93c8214faf28cea0362cbf2e5c7736f)